pages/dkp/kaptain/2.0.0/configuration/index.md

```diff
@@ -10,53 +10,137 @@ enterprise: false
+| ingress.customDomainName | string | `""` | Custom domain name used to access Kaptain, for example, kaptain.mycluster.company.com |
+| ingress.base64CustomCertificate | string | `""` | base64-encoded contents of a custom certificate file (.crt) to use with the provided custom domain name |
+| ingress.base64CustomCertificateKey | string | `""` | base64-encoded contents of a custom certificate private key file (.key) to use with the provided custom domain name |
+| ingress.namespace | string | `"kaptain-ingress"` | Namespace to install Kaptain Ingress resources |
+| core.enabled | bool | `true` |  |
+| core.namespace | string | `"kubeflow"` |  |
+| core.notebook.defaultImage | string | `"mesosphere/kubeflow-dev:b66cecc3-jupyter-spark-3.0.0-horovod-0.24.2-tensorflow-2.8.0"` | Default image to use when creating a new notebook server |
+| core.notebook.images[0] | string | `"mesosphere/kubeflow-dev:b66cecc3-jupyter-spark-3.0.0-horovod-0.24.2-tensorflow-2.8.0"` | JupyterLab with Tensorflow, Spark and Horovod pre-installed |
+| core.notebook.images[1] | string | `"mesosphere/kubeflow-dev:c15bf3fa-jupyter-spark-3.0.0-horovod-0.24.2-tensorflow-2.8.0-gpu"` | JupyterLab with Tensorflow, CUDA, Spark and Horovod pre-installed with GPU support |
+| core.notebook.images[2] | string | `"mesosphere/kubeflow-dev:22caa41c-jupyter-spark-3.0.0-horovod-0.24.2-pytorch-1.11.0"` | JupyterLab with Pytorch, Spark and Horovod pre-installed |
+| core.notebook.images[3] | string | `"mesosphere/kubeflow-dev:d431dd5f-jupyter-spark-3.0.0-horovod-0.24.2-pytorch-1.11.0-gpu"` | JupyterLab with Pytorch, CUDA, Spark and Horovod pre-installed with GPU support |
+| core.notebook.images[4] | string | `"mesosphere/kubeflow-dev:44625480-jupyter-spark-3.0.0-horovod-0.24.2-mxnet-1.9.0"` | JupyterLab with MXNet, Spark and Horovod pre-installed |
+| core.notebook.images[5] | string | `"mesosphere/kubeflow-dev:e6c6aec9-jupyter-spark-3.0.0-horovod-0.24.2-mxnet-1.9.0-gpu"` | JupyterLab with Tensorflow, CUDA, Spark and Horovod pre-installed with GPU support |
+| core.notebook.tolerationGroups | list | `[]` | Pod toleration group configurations for Notebook servers |
+| core.notebook.affinityConfig | list | `[]` | Pod affinity configurations for Notebook servers |
+| core.notebook.enableCulling | bool | `false` | Enables scale down idling notebooks to freeing up the allocated resources. |
+| core.notebook.idleTimeMinutes | int | `1440` | The period of time a notebook can stay idle before it gets culled (scaled down) |
+| core.notebook.cullingCheckPeriodMinutes | int | `1` | Notebook status check period |
+| core.disableAntiAffinity | bool | `false` | Enables single-node installation. NOT FOR PRODUCTION USE! Disabling anti-affinity allows installing Kaptain on a smaller number of nodes (<3) or on a single node. This installation mode must be used for evaluation purposes only. Not recommended for production use! No data integrity guarantees in case of a hardware failure! |
```

Comment:
```suggestion
| core.disableAntiAffinity | bool | `false` | Enables single-node installation. **DO NOT USE IN PRODUCTION ENVIRONMENTS!** Disabling anti-affinity allows installing Kaptain on a smaller number of nodes (less than 3) or on a single node. This installation mode must be used **for evaluation purposes only**. No data integrity guarantees in case of a hardware failure! |
```